src/linter/linter.ts

```diff
@@ -163,7 +163,7 @@ async function lint(
 		reader,
 	});
 
-	const res = await lintWorkspace(workspace, filePathsWorkspace, options, config, matchedPatterns);
+	const res = await lintWorkspace(workspace, filePathsWorkspace, options, config, ignorePattern, matchedPatterns);
```

Comment:
The whole build ignorePattern does not need to be provided in lintWorkspace.
The configuration is also available there. The same has already been done for the filePatterns there: https://github.com/SAP/ui5-linter/blob/06871b660db5ed01799abff8558cd4f8f59dc835/src/linter/lintWorkspace.ts#L27